peachjam/views/legislation.py

```diff
@@ -12,6 +12,10 @@ class LegislationListView(FilteredDocumentListView):
     context_object_name = "documents"
     paginate_by = 20
 
+    def get_queryset(self):
+        queryset = super(LegislationListView, self).get_queryset()
```

Comment:
Push this down into FilteredDocumentListView, so that all list views get it. You can also simplify it a bit:

```python
def get_queryset(self):
    return super().get_queryset().order_by("-date")
```

Reply:
The `super(LegislationListView, self)` is the old way of calling a superclass's method, which is no longer required and is simplified to just `super()`.

Reply:
done